lib/services/local/access_list.go

```diff
@@ -361,18 +362,17 @@ func (a *AccessListService) DeleteAllAccessListMembers(ctx context.Context) erro
 	return trace.Wrap(a.memberService.DeleteAllResources(ctx))
 }
 
+func getName[R types.Resource](r R) string {
```

Comment:
There are multiple copies of this function in various packages. Time to go somewhere more central?